CHANGELOG.rst

```diff
@@ -1,5 +1,60 @@
 .. _changelog:
 
+0.6.3
+-----
+
+New feature
+~~~~~~~~~~~~
+
+⭐️ Project details now include a listing of the commit history
+
+Notable improvements
+~~~~~~~~~~~~~~~~~~~~
+
+🚄 Environments: auto-saved branches are filtered per username
+
+🚄 Improve markdown rendering and code highlighting
+
+🚄 Editing markdown files is easier as ``ckeditor`` is partially integrated inside Renku
+
+Bug fixes
+~~~~~~~~~
+
+* Dataset contains all folders from unzipped file
+* Failing to retrieve metadata for one dataset does not cause the others to fail
+* Improved UX for when datasets take too long
+* Datasets: no failure when adding ignored files
+
+Miscellaneous
+^^^^^^^^^^^^^
+
+- Default R templates now use the latest R (4.0.0); `bioc 3_11 <https://www.bioconductor.org/news/bioc_3_11_release>`__ also now available
```

Comment:
maybe link to latest R: `latest R (4.0.0) <https://github.com/SwissDataScienceCenter/renkulab-docker#r>`_
and use this for bioc: https://github.com/SwissDataScienceCenter/renkulab-docker#bioc

Reply:
I'm not sure I want to point users to the Dockerfile sources. I would rather put a link to the docker images instead.

Reply:
yeah but if users go to the bioc link they won't know how to add the one we built. we can put `renku/renkulab-bioc:RELEASE_3_11-renku0.10.4-0.6.3` ?

Reply:
yes, that makes more sense. I'll put that for both bioc and R